usage/basics.md

```diff
@@ -4,7 +4,7 @@
 
 ```dart
 FlutterMap(
-    controller: MapController(),
+    mapController: MapController(),
```

Comment:
@JaffaKetchup Should this line be included as well?

Reply:
Yes. Sorry, missed this one!